install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.json

```
@@ -136,6 +136,7 @@
           "envoy.extensions.upstreams.http.v3.HttpProtocolOptions": {
             "@type": "type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions",
             "commonHttpProtocolOptions": {
+              "idleTimeout": "{{ .Values.envoy.idleTimeoutDurationSeconds }}s",
```

This and the other usages in this file set up the bootstrap's idleTimeouts for static clusters like `egress-cluster`, do we need to port this to the Daemonset bootstrap as well?

Could be a followup PR although it is better UX to have these clusters have a lower timeout, so that they'll fail faster if something goes badly wrong.